### PR TITLE
feat: Employee Referral

### PR DESCRIPTION
**Pull request by Anurag810**
image

### CODE REVIEW

File: erpnext_documentation/www/docs/user/manual/en/human-resources/employee-referral.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+<img class="screenshot" alt="Leave Allocations"
+	src="{{docs_base_url}}/assets/img/human-resources/create-job-applicant.png">
+
+On click, Job Applicant will be created with status "Open" and the status of Employee Referral document will change to "In Process"
+
+If someone changes the status of the Job Applicant document to "Hold" or "Replied", The status of the Employee Referral document will remain "In Process". If the status of the Job Applicant is "Accepted" or "rejected", The status of the Employee Referral document will also change to "Accepted" or "rejected" respectively.
+
+### 3.2 Is applicable for referral bonus
+Many companies provides Bonus to there employees for such referral. ERPNext allow you pay bonus to employee for there referral.
```
Comment (asbasawaraj) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/employee-referral.md:45]: "Many companies provide bonuses to their employees for such referrals. ERPNext allows you to track the payment of the bonus to employees for their referral."

File: erpnext_documentation/www/docs/user/manual/en/human-resources/employee-referral.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+For Referral bonus you need to check **Is applicable for referral bonus** checkbox before submitting the document. After submitting the document Create Additional Salary Button will Appear at top right corner, if status is Accepted.
```
Comment (asbasawaraj) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/employee-referral.md:47]: "For the Referral bonus, you need to check the **Is applicable for referral bonus** checkbox before submitting the document. After submitting the document Create Additional Salary **b**utton will Appear at the top right corner if the status is "Accepted"."

**Review by asbasawaraj — CHANGES_REQUESTED** [review]
Please check capitalization as suggested in remarks

**Comment by barredterra** [comment]
@Anurag810 docs have been refactored to have a v12 and v13 folder for the respective version of ERPNext. Kindly update your PR to take this new structure into account.

**Comment by Anurag810** [comment]
@asbasawaraj Fixed changes

File: erpnext_documentation/www/docs/v13/user/manual/en/human-resources/employee-referral.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+To access Employee Referral, go to:
+
+> Human Resources > Recruitment > Employee Referral
+
+
+## 1. Prerequisites
+1. [Employee](/docs/user/manual/en/human-resources/employee)
+1. [Additional Salary](/docs/user/manual/en/human-resources/additional-salary)
+1. [Job Applicant](/docs/user/manual/en/human-resources/job-applicant)
```
Comment (barredterra) [inline erpnext_documentation/www/docs/v13/user/manual/en/human-resources/employee-referral.md:20]: Please refactor these links to point to the correct version (v13).

File: erpnext_documentation/www/docs/v13/user/manual/en/human-resources/employee-referral.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+1. Save and Submit.
+
+
+<img class="screenshot" alt="Leave Allocations"
+    src="{{docs_base_url}}/assets/img/human-resources/employee-referral.png">
+
+## 3. Features
+
+### 3.1 Creating Job Applicant and auto-syncing status from Job Applicant
+When you submit an employee referral document the initial status will be "Pending". After Submitting the document Create Job Applicant Button will Appear at the top right corner.
```
Comment (barredterra) [inline erpnext_documentation/www/docs/v13/user/manual/en/human-resources/employee-referral.md:35]:
```suggestion
When you submit an employee referral document the initial status will be "Pending". After submitting the document, the "Create Job Applicant" button will appear at the top right corner.
```

File: erpnext_documentation/www/docs/v13/user/manual/en/human-resources/employee-referral.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+On clicking, Job Applicant will be created with the status "Open" and the status of Employee Referral document will change to "In Process"
```
Comment (barredterra) [inline erpnext_documentation/www/docs/v13/user/manual/en/human-resources/employee-referral.md:37]:
```suggestion
Clicking this button will create a new **Job Applicant** with the status "Open". The status of the **Employee Referral** document will change to "In Process"
```

File: erpnext_documentation/www/docs/v13/user/manual/en/human-resources/employee-referral.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+<img class="screenshot" alt="Leave Allocations"
+    src="{{docs_base_url}}/assets/img/human-resources/create-job-applicant.png">
+
+If someone changes the status of the Job Applicant document to "Hold" or "Replied", The status of the Employee Referral document will remain "In Process". If the status of the Job Applicant is "Accepted" or "Rejected", The status of the Employee Referral document will also change to "Accepted" or "Rejected" respectively.
```
Comment (barredterra) [inline erpnext_documentation/www/docs/v13/user/manual/en/human-resources/employee-referral.md:42]:
```suggestion
When someone changes the status of the **Job Applicant** to "Hold" or "Replied", the status of the **Employee Referral** will remain "In Process". If the status of the **Job Applicant** changes to "Accepted" or "Rejected", the status of the **Employee Referral** document will also change to "Accepted" or "Rejected" respectively.
```

File: erpnext_documentation/www/docs/v13/user/manual/en/human-resources/employee-referral.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+### 3.2 Managing referral bonus
+Many companies provide bonuses to their employees for such referrals. ERPNext allows you to track the payment of the bonus to employees for their referral.
```
Comment (barredterra) [inline erpnext_documentation/www/docs/v13/user/manual/en/human-resources/employee-referral.md:45]:
```suggestion
Many companies provide bonuses to their employees for such referrals. ERPNext allows you to track the payment of the bonus to the employee for their referral.
```

File: erpnext_documentation/www/docs/v13/user/manual/en/human-resources/employee-referral.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+For the Referral bonus, you need to check the Is applicable for referral bonus checkbox before submitting the document. After submitting the document Create Additional Salary button will Appear at the top right corner of the status is "Accepted".
```
Comment (barredterra) [inline erpnext_documentation/www/docs/v13/user/manual/en/human-resources/employee-referral.md:47]:
```suggestion
For the Referral bonus, you need to check the "Is applicable for referral bonus" checkbox before submitting the document. After submitting the document, the "Create Additional Salary" button will appear at the top right corner, if the status is "Accepted".
```

**Review by barredterra — CHANGES_REQUESTED** [review]

File: erpnext_documentation/www/docs/v13/user/manual/en/human-resources/employee-referral.md (hunk 3 above)
Comment (Anurag810) [reply erpnext_documentation/www/docs/v13/user/manual/en/human-resources/employee-referral.md:20]: Done

**Comment by Anurag810** [comment]
@asbasawaraj  @barredterra Fixed suggested changes, Please merge.

**Review by barredterra — APPROVED** [review]

**Pull request merged**